### PR TITLE
Feature/translate toast

### PR DESCRIPTION
**Pull request by LorenzoJokhan**
# Description

Please include
- a summary of the changes
- relevant motivation and context
- a list of any dependencies that are required for this change

## Issue reference

Fixes # (issue)

## Type of change

Is it a new feature, bug fix, code improvement, etc.  
If it is a breaking change what needs to be done to fix that

## Documentation

Is the documentation updated, maybe a link

## Tests

(How) has the change been tested

## Branch

If the branch to merge to is not development

### CODE REVIEW

File: packages/cms/lib/modules/openstad-assets/public/css/toast.less [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  } 
+
+.toast-container > p {
+    display: block;
+    padding: 1rem 1.5rem;
+
+    color: white;
+    border: 1px solid lightgrey;
+    border-radius: .5rem;
+    filter: drop-shadow(0px 4px 8px rgba(0, 0, 0, 0.08)) drop-shadow(0px 2px 4px rgba(0, 0, 0, 0.08));}
```
Comment (rudivanhierden) [inline packages/cms/lib/modules/openstad-assets/public/css/toast.less]:
```suggestion
    filter: drop-shadow(0px 4px 8px rgba(0, 0, 0, 0.08)) drop-shadow(0px 2px 4px rgba(0, 0, 0, 0.08));
}
```

File: packages/cms/lib/modules/openstad-global/public/js/always.js [hunk 2]
```diff
@@ -62,3 +69,35 @@ handleNode = function (node, toBeTranslated) {
...
+
+function cleanupToasts(container) {
+    if(container) {
+        container.childNodes.forEach(function(pElement){
+            setTimeout(function(){
+                container.removeChild(pElement);
+            }, 3000);
+        });
+    }
+}
```
Comment (rudivanhierden) [inline packages/cms/lib/modules/openstad-global/public/js/always.js]: Where is this method used?

File: packages/cms/lib/modules/translation-widgets/public/js/always.js [hunk 3]
```diff
@@ -69,17 +73,20 @@ apos.define('translation-widgets', {
                         sentences = sentences.map(function(sentence) { return sentence.text });
                         changeTextInNodes(sentences, nodes);
                         setSelectEnabled(select);
+                        addToast(toastContainer, "success", "De pagina is succesvol vertaald", 5000);
                     }, 
-                    error: function() {
+                    error: function(e) {
```
Comment (rudivanhierden) [inline packages/cms/lib/modules/translation-widgets/public/js/always.js]: The error `e` isn't used in this method, so I think we can leave it out?

**Review by rudivanhierden — COMMENTED** [review]
Looking good so far! Just a few questions (and one nitpicky formatting comment)

File: packages/cms/lib/modules/openstad-assets/public/css/toast.less (hunk 1 above)
Comment (LorenzoJokhan) [reply packages/cms/lib/modules/openstad-assets/public/css/toast.less]: Done

File: packages/cms/lib/modules/translation-widgets/public/js/always.js (hunk 3 above)
Comment (LorenzoJokhan) [reply packages/cms/lib/modules/translation-widgets/public/js/always.js]: e weggehaald

File: packages/cms/lib/modules/openstad-global/public/js/always.js [hunk 4]
```diff
@@ -9,7 +9,12 @@ apos.on('ready', function () {
      */    
     var translationWidgetOnSamePage = $('.translation-widget-select').length > 0;
 
-    if (!translationWidgetOnSamePage && selectedLanguage && selectedLanguage !== 'nl') {
+    var userHasSpecialRole = self.openstadUserRole === 'admin' || self.openstadUserRole === 'moderator' || self.openstadUserRole === 'editor';
```
Comment (rudivanhierden) [inline packages/cms/lib/modules/openstad-global/public/js/always.js]: I think we can use `hasModeratorRights` global variable set in layout.html, (`self.hasModeratorRights` I think in this context?)

File: packages/cms/lib/modules/translation-widgets/public/js/always.js [hunk 5]
```diff
@@ -89,5 +96,12 @@ function setSelectedLanguage(language) {
  * collecting the initial values and if the language is not the default 'nl', fetching the translations
  */
 apos.on('ready', function() {
-    setSelectedLanguage(sessionStorage.getItem('targetLanguageCode'));
+    var select = document.querySelector('.translation-widget-select');
+
+    var isNormalUser = !(self.openstadUserRole === 'admin' || self.openstadUserRole === 'moderator' || self.openstadUserRole === 'editor');
```
Comment (rudivanhierden) [inline packages/cms/lib/modules/translation-widgets/public/js/always.js]: Same goes for this check

**Review by rudivanhierden — CHANGES_REQUESTED** [review]

**Review by rudivanhierden — APPROVED** [review]

**Pull request merged**